skins/oasis/css/core/page.scss

```diff
@@ -281,6 +278,61 @@
 	}
 }
 
+#WikiaRecentActivity {
+	background-color: transparent;
+	border: none;
+	padding: 20px 0;
+
+	.activity-module-header {
+		border-bottom: $color-page-border solid 1px;
```

Comment:
The proper format is `width style color`. https://developer.mozilla.org/en/docs/Web/CSS/border-bottom